tests/foreman/cli/test_contentviews.py

```diff
@@ -154,23 +153,13 @@ def test_cv_create_cli(self, test_data):
         @assert: content views are created
 
         """
-        org_obj = make_org(cached=True)
-
-        result = Org.info({'id': org_obj['id']})
-        self.assertEqual(result.return_code, 0, "Failed to create object")
-        self.assertEqual(
-            len(result.stderr), 0, "There should not be an exception here")
-
-        test_data['organization-id'] = org_obj['id']
-
         try:
-            con_view = make_content_view(test_data)
+            test_data['organization-id'] = make_org(cached=True)['id']
+            content_view = make_content_view(test_data)
```

Comment:
The method ``make_content_view`` takes a dictionary but we're passing only an ID?